workflow/scripts/validate_data.py

```diff
@@ -190,7 +188,10 @@ def get_buses(n):
     buses_clean = [ba.split("0")[0] for ba in buses]
     buses_clean = [ba.split("-")[0] for ba in buses_clean]
     buses = list(OrderedDict.fromkeys(buses_clean))
-    buses.pop(1)
+    if "ERCO" in buses:
```

Comment:
@nomineb Hey Nomin, FYI I had to add this line to fix for Texas.... keep an eye out to make sure that your scripts can generalize between different interconnect wildcards.